src/herder/Upgrades.cpp

```diff
@@ -994,25 +1172,60 @@ upgradeFromProtocol15To16(AbstractLedgerTxn& ltx)
     }
 }
 
+#ifdef ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
+static void
+createConfigSettingEntry(ConfigSettingEntry const& configSetting,
+                         AbstractLedgerTxn& ltxRoot)
+{
+    LedgerEntry e;
+    e.data.type(CONFIG_SETTING);
+    e.data.configSetting() = configSetting;
+    LedgerTxn ltx(ltxRoot);
+    ltx.create(e);
+    ltx.commit();
+}
+
+static void
+initializeConfigs(AbstractLedgerTxn& ltx)
+{
+    ConfigSettingEntry contractMaxSize(CONFIG_SETTING_CONTRACT_MAX_SIZE_BYTES);
+    contractMaxSize.contractMaxSizeBytes() = 65536;
```

Comment:
there should be an actual constant somewhere

Reply:
Added a constant